CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Command Generate - Add logs to requests on the project generated.
+
+### Changed
+- Command Generate - Update comments in routes.
```

Comment:
I think this is not required since it's an internal change of v3.

Reply:
Yeah but that was not released with the previous PR so I still think it should be here 😉